app/metrics.yaml

```diff
@@ -55,13 +55,51 @@ events:
     extra_keys:
       search_suggestion:
         description: "A boolean that tells us whether or not the search term was suggested by the Awesomebar"
+      search_shortcut:
+        description: "A boolean that tells us whether or not the search was conducted through a search shortcut"
     bugs:
     - 959
     data_reviews:
     - https://github.com/mozilla-mobile/fenix/pull/1067#issuecomment-474598673
     notification_emails:
     - telemetry-client-dev@mozilla.com
     expires: "2020-03-01"
+  ss_menu_opened:
+    type: event
+    description: >
+      A user opened the search shortcut menu in the search view by pressing the shortcuts button
+    bugs:
+    - 793
+    data_reviews:
+    - https://github.com/mozilla-mobile/fenix/pull/1202#issuecomment-476870449
+    notification_emails:
+    - telemetry-client-dev@mozilla.com
```

Comment:
Still need to update this so your team will actually receive expiry reminders. @vesta0 do you have an email for this?